src/Traits/VoyagerUser.php

```diff
@@ -13,7 +13,7 @@
 {
     public function role()
     {
-        return $this->belongsTo(Voyager::modelClass('Role'));
```

Comment:
We should leave this alone.  It functions identically since you changed it to the reciprocal relation, but it's cleaner before.